src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl

```diff
@@ -57,6 +57,20 @@
               "format": "uuid"
             }
           },
+          {
+            "in": "header",
+            "name": "x-client-id-from-client",
```

Comment:
This header should not be exposed but is internal to the service

Reply:
this header is valued using the client id query parameter injected by payment request and then passed back from f.e. into POST transaction api call so that b.e. can save this information.

This have been done since in POST transactions coming from a cart no information about original cart id is passed to transactions-service so the other way can be to add carts reading capability to transactions service that can retrieve the client id associated to that cart